tools/releaser/scripts/breaking-changes.sh

```diff
@@ -12,23 +11,36 @@ script_path=$(dirname "$0")
 echo "Installing go-apidiff"
 go install github.com/joelanford/go-apidiff@latest > /dev/null
 
-GIT_BASE_REF=${GIT_BASE_REF:-git rev-parse head || echo}
+current_ref=$(git rev-parse main || echo)
+GIT_BASE_REF=${GIT_BASE_REF:-$current_ref}
+TARGET_BREAKING_CHANGES_FILE=${TARGET_BREAKING_CHANGES_FILE:""}
 
 echo "Running breaking changes check for $GIT_BASE_REF"
 
 pushd "$script_path/../../../" || exit ## workaround for --repo-path="../" not working
-BREAKING_CHANGES=$("$GOPATH/bin/go-apidiff" "$GIT_BASE_REF" --compare-imports="false" --print-compatible="false")
-popd || exit
+echo "Changed directory to $(pwd)"
+set +e
+BREAKING_CHANGES=$("$GOPATH/bin/go-apidiff" "$GIT_BASE_REF" --compare-imports="false" --print-compatible="false" )
+set -e
+#popd || exit
```

Comment:
leftover?

Reply:
Wanted to test if we need it but let's keep it for sake of using this scripts from different folders.

Reply:
Resolved